packages/expo-video/src/VideoPlayer.types.ts

```diff
@@ -193,7 +195,9 @@ export declare class VideoPlayer extends SharedObject<VideoPlayerEvents> {
   replace(source: VideoSource): void;
 
   /**
-   * Seeks the playback by the given number of seconds.
+   * Seeks the playback by the given number of seconds. The time to which the player seeks may differ from the specified requested time for efficiency,
+   * depending on the encoding and what is currently buffered by the player. Use this function to implement playback controls that seek by specific amount of time,
+   * in which case the actual time usually does not have to be precise. For frame accurate seeking use the [`currentTime`](#currenttime) property.
```

Comment:
```suggestion
   * in which case, the actual time usually does not have to be precise. For frame accurate seeking, use the [`currentTime`](#currenttime) property.
```
Let's need to add those pauses in the above so that the natural flow of readability is maintained.